rte/pkg/sysinfo/sysinfo.go

```diff
@@ -50,14 +58,47 @@ func ResourceMappingFromString(s string) map[string]string {
 		}
 		items := strings.SplitN(keyvalue, "=", 2)
 		if len(items) != 2 {
-			klog.Infof("malformed resource mapping item %q, skipped", keyvalue)
+			klog.Warningf("malformed resource mapping item %q, skipped", keyvalue)
 			continue
 		}
 		rmap[strings.TrimSpace(items[0])] = strings.TrimSpace(items[1])
 	}
 	return rmap
 }
 
+func ReservedMemoryFromString(s string) map[int]int64 {
```

Comment:
It can be a good idea to have unittests for this method.

Reply:
good idea indeed, done!